### ISSUE TITLE
Custom place holders for value ONE.OF

### ISSUE THREAD

**Issue by officiallysameer** [authorjapps/zerocode#67]
Something like below
e.g. 
Currently framework supports the below.
Response:
```
{
   "currentStatus": "Searching"
}
```

In certain use-cases, the `"currentStatus"` can be one-of multiple values.
Then it would be nice to support like below.
Assertions:
```
{
       "currentStatus": "$ONE.OF:[Found, Searching, Not Looking]"
}
```
That means when the actual response with `currentStatus` having "Found" or "Searching" or "Not Looking" will pass.

**Comment by sarthak1997** [comment]
Please clarify my understanding of this issue, is it like rather than hard-coding the placeholder values like "Found" or "Searching" in ONE.OF, the placeholder values will be given through config or something, so while the json gets rendered those placeholder values will be picked from the config? Please correct me if i am wrong. Thanks!

**Comment by authorjapps** [comment]
@sarthak1997 , just updated the description now with more details.

**Comment by sarthak1997** [comment]
Oh okay, thanks @authorjapps for the clarification. Yes, I am picking this ticket.

**Comment by authorjapps** [comment]
Cool!

Sample code for existing `"$CONTAINS.STRING:"`
> ...zerocode.core.engine...ZeroCodeJsonTestProcesorImpl#ASSERT_VALUE_CONTAINS_STRING

```java
...
} else if (value instanceof String && ((String) value).startsWith(ASSERT_VALUE_CONTAINS_STRING)) {
        String expected = ((String) value).substring(ASSERT_VALUE_CONTAINS_STRING.length());
        asserter = new FieldHasSubStringValueAsserter(path, expected);
...
```

Basically you need to follow the above similar pattern 👍

**Comment by authorjapps** [comment]
@sarthak1997 , did you get chance to impl this?

**Comment by jbonn360** [comment]
I can work on this unless @sarthak1997 still is

**Comment by jbonn360** [comment]
@authorjapps I'd like to work on this issue if it's ok

**Comment by authorjapps** [comment]
@jbonn360 , thanks mate! 

@sarthak1997, FYI only- You can still help us in reviewing the PR, please!

***
_The below is not part of this ticket._

Just thinking the best way we can solve this, if any of the values contains a comma(`,`)

Vanila case:
> "currentStatus": "$ONE.OF:[Found, Searching, Not Looking]"

<br/>

Comma case(then we can go for pipe `|` splitter ):
> "currentStatus": "$ONE.OF:[Foo,Bar | Foo1,Bar1 | Foo2,Bar2]"

There is no such usecase logged yet, but we will raise a separate TCH-DEBT ticket to handle this.

**Comment by jbonn360** [comment]
@authorjapps understood, I'll raise the PR for the vanilla test case for now. I wouldn't mind working on the Comma case as well once the ticket is raised.

**Comment by authorjapps** [comment]
Feature available in version 1.3.9 and higher.
